au4/src/projectscene/qml/Audacity/ProjectScene/clipsview/ClipsSelection.qml

```diff
@@ -15,13 +16,8 @@ Item {
     }
 
     function onSelectionChanged(p1, p2) {
-        if (p2.x > p1.x) {
-            selRect.x = p1.x
-            selRect.width = p2.x - p1.x
-        } else {
-            selRect.x = p2.x
-            selRect.width = p1.x - p2.x
-        }
+        selRect.x = root.context.timeToPosition(root.context.selectionStartTime)
+        selRect.width = root.context.timeToPosition(root.context.selectionEndTime) - selRect.x
```

Comment:
it looks like these should be context methods or properties, something like that: 
```
root.context.selectionStartPosition()
root.context.selectionEndPosition()
```
or better as properties 

```
root.context.selectionStartPosition
root.context.selectionEndPosition
```